src/components/tabs/__internal__/tab-title/tab-title.style.ts

```diff
@@ -523,6 +569,10 @@ const tabTitleStyles = css<
         background-color: var(--colorsActionMinor200);
       `}
     `}
+
+    & span[data-component="icon"] {
```

Comment:
**suggestion(non-blocking)**: given the style rules are tied to `Icon` component, what do you think about using a component selector here? 

This might help make the relationship between tab title and `Icon` more explicit:

```suggestion
    & ${StyledIcon} {
```